platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Menu.kt

```diff
@@ -824,7 +852,7 @@ internal fun MenuSubmenuItem(
             }
         }
 
-        if (itemState.isSelected) {
+        if (isSelected) {
```

Comment:
This change looks sus. Looks like we have two different "isSelected" values now, and only one is used

Reply:
Merged the itemState with the parameter